### PR TITLE
Ep/add focalnet tiny lrf in1k

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/focalnet_tiny_in1k_submission/model.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import timm
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+"""
+Template module for a base model submission to brain-score
+"""
+
+
+def get_model_list():
```
Comment (mike-ferguson) [inline brainscore_vision/models/focalnet_tiny_in1k_submission/model.py:17]: can remove `get_model_list`

File: brainscore_vision/models/focalnet_tiny_in1k_submission/requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+timm
```
Comment (mike-ferguson) [inline brainscore_vision/models/focalnet_tiny_in1k_submission/requirements.txt:1]: needs 

```
torch
torchvision
```

**Review by mike-ferguson — CHANGES_REQUESTED** [review]
Can remove `get_model_list`, add two more needed packages to `requirements.txt`

**Comment by pellegreene** [comment]
Made the requested changes!

**Pull request merged**